crates/snp_validation/src/lib.rs

```diff
@@ -39,9 +45,13 @@ flags! {
         SMT = 1 << 0,
     }
 
+    /// These items are mutually exclusive
```

Comment:
This comment is out of context, first you should explain what it *is* and only then describe semantics

Reply:
Fixed